server/test/test_scanpy_engine.py

```diff
@@ -98,5 +98,9 @@ def test_schema_produces_error(self):
         with pytest.raises(TypeError):
             self.data._create_schema()
 
+    def test_config(self):
+        self.assertEqual(self.data.features["layout"]["obs"], {'available': True, 'interactiveLimit': 15000})
```

Comment:
this magic value is in the code twice. consider making it (and all the other interactiveLimit values) constants.

Reply:
I'm going to do that when I do the plugin architecture and the limits are actually real. Right now they are mostly just placeholders